libtock-sync/services/unit_test.h

```diff
@@ -86,12 +86,12 @@ extern "C" {
  * for FAIL.
  */
 typedef struct unit_test_fun_s {
-    bool (*fun)(void);
-    char name[24];
+  bool (*fun)(void);
+  char name[24];
 } unit_test_fun;
 
 #define TEST(NAME) { \
-    test_##NAME,     \
```

Comment:
I _assume_ this doesn't change anything behaviorally, but C-macro magic like this always makes me want to look twice

Reply:
Added `sp_pp_concat                    = remove`